### ISSUE TITLE
[Feature request] Connect to device

### ISSUE THREAD

**Issue by michalmielczynski** [xou816/spot#571]
I'm loving Spot, its' integration with system, speed and features and of course being Open Source.

As this is early version of app, I suppose there's a lot to improvements planned. I'm even look at the source code but since I'm completely new to Rust and GTK development, this is not going to be very helpful soon.

Anyways, THE FEATURE REQUEST: I wonder if its possible to get the 'Connect to device' as we see in Spotify application. 
In short, you can play songs on remote devices and use the laptop as remote control device.

I'm sure you know the feature - the question is if that's even possible in decent time/effort in Spot?
Any plans?

**Comment by ondras12345** [comment]
Duplicate of https://github.com/xou816/spot/issues/55

**Comment by xou816** [comment]
Thanks @ondras12345  :)
